doc/design/README.md

```diff
@@ -29,4 +28,4 @@ This is an example how to render PlantUML in Github:
 
 workflow using svg file:
 
-![plantuml github workflow](diagrams/puml_github_workflow.svg)
+![plantuml github workflow](../website/images/puml_github_workflow.svg)
```

Comment:
This document will be included with MkDocs on iceoryx.io, right? Please use the following format for link resolution then (plugin which resolves the links is `mkdocs-autolinks-plugin`):

```suggestion
![plantuml github workflow](puml_github_workflow.svg)
```

Reply:
Got your point and agree. However, I'm not sure if you read my first sentence, which is a question:

> This document will be included with MkDocs on iceoryx.io, right?

I see two possible solutions:

1) Either define for each `.md` document if it is read via [iceoryx.io](https://iceoryx.io) or via GitHub's web GUI
   Dual-use would be limited only to the `README.md`s of the examples to avoid duplication
2) Create some magic scripts with regular expression that extract the filename e.g. using `basename` before running `mkdocs serve`

Do you have any other nifty ideas?

Reply:
A clean solution would in my opinion be a script that adjusts the links. But we do not have enough time for it so I would in the design section of the website just link to github.

Reply:
@elfenpiff Again, you didn't answer my question :grin: 

> This document will be included with MkDocs on iceoryx.io, right?

If I got it right, the answer to the question is "No" and hence we can resolve this conversation and merge this PR. This document here will solely be used on GitHub and not be included on iceoryx.io.

However, for the individual design documents I would appreciate a lot if we find a solution before the `v2.0` release. We definitely should add the individual articles to iceoryx.io. The reason is we want to avoid code duplication and just symlink
like:

```
doc/design/website/design/foo-article.md -> ../../design/foo-article.md
```

Reply:
@mossmaurice My answer is no. The website can wait for now.